File: go/kata/game_test.go
package kata

import "testing"

func TestGutterGame(t *testing.T) {
	var game Game = Game {}
	for i := 0; i < 20; i++ {
		game.roll(0)
	}
	if game.score() != 0 {
		t.Errorf("Game.score() for a 0 pins game expect 0, got %d", game.score()) 
	}
}

File: ruby/kata/game_test.rb
require 'test/unit'
require_relative 'game' 

class GameTest < Test::Unit::TestCase

  def test_gutter_game
    game = Game.new
    20.times do
      game.roll(0)
    end
    assert_equal 0, game.score
  end

  def test_all_ones
    game = Game.new
    20.times do
      game.roll(1)
    end
    assert_equal 20, game.score
  end
end
